src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
```

Comment:
You can also add a script to insert 2-3 default books to make it easier to test new features in the future